homeassistant/components/roborock/strings.json

```diff
@@ -337,44 +511,6 @@
       },
       "washing_left": {
         "name": "Washing left"
-      },
-      "zeo_error": {
-        "name": "Error",
-        "state": {
-          "communication_error": "Communication error",
-          "door_lock_error": "Door lock error",
-          "drain_error": "Drain error",
-          "drying_error": "Drying error",
-          "drying_error_e_12": "Drying error E12",
-          "drying_error_e_13": "Drying error E13",
-          "drying_error_e_14": "Drying error E14",
-          "drying_error_e_15": "Drying error E15",
-          "drying_error_e_16": "Drying error E16",
-          "drying_error_restart": "Restart the washer",
-          "drying_error_water_flow": "Check water flow",
-          "heating_error": "Heating error",
-          "inverter_error": "Inverter error",
-          "none": "[%key:component::roborock::entity::sensor::vacuum_error::state::none%]",
-          "refill_error": "Refill error",
-          "spin_error": "Re-arrange clothes",
-          "temperature_error": "Temperature error",
-          "water_level_error": "Water level error"
-        }
-      },
-      "zeo_state": {
```

Comment:
I realize i told you to rename to not have "zeo" prefix but the existing key is using it. can you revert this so we don't lose translations?

Reply:
I think my comment may have been a bad idea for some of the fields that have unique values to these specific devices, like the mode etc